schema.sql

```diff
@@ -8,3 +8,7 @@ CREATE TABLE animals (
     neutered boolean NOT NULL,
     weight_per_kg decimal NOT NULL
 );
+
+/* Alter animals table and add new column*/
+
+ALTER TABLE animals ADD COLUMN species varchar;
```

Comment:
- [ ] Kindly fix this error, when I am trying to insert your data this is throwing an error because the id is null, this happens because you did not put the id when inserting data. You can solve this by putting the id in your queries or just auto-increment your Id. Check this doc on how to [auto-increment the id using serial](https://www.geeksforgeeks.org/postgresql-create-auto-increment-column-using-serial/). This is :point_down:  the error I am getting.

![image](https://user-images.githubusercontent.com/65068771/154463254-4aded2cd-20a9-4b42-945f-79e3d822a7ed.png)

Reply:
Thank you for the review @andersonlebon

Reply:
Done